component/distributed_cache/distributed_cache.go

```diff
@@ -689,6 +714,11 @@ func (dc *DistributedCache) CloseFile(options internal.CloseFileOptions) error {
 			log.Err("DistributedCache::SyncFile : Failed to ReleaseFile for Azure file : %s", options.Handle.Path)
 		}
 	}
+
+	if options.Handle.IsFsDebug() {
```

Comment:
// Debug is exclusive, if debug is set dcache and azure flags cannot be set.
common.Assert(!options.Handle.IsFsDebug() || (!options.Handle.IsFsDcache() && !options.Handle.IsFsAzure()))